scripts/lint.rust.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -euo pipefail
+# Lint the rust code
+cargo clippy --locked --target wasm32-unknown-unknown --all-features -- -D warnings -W clippy::pedantic -A clippy::module-name-repetitions -A clippy::struct-field-names
```

Comment:
there is `npm run clippy` declared in `package.json`, can you also update this shorthand?

I use it when I develop locally.

Reply:
Done